modules/openapi-generator/src/main/resources/rust/reqwest/api.mustache

```diff
@@ -2,7 +2,8 @@
 
 use reqwest;
 
-use crate::apis::ResponseContent;
+use crate::apis::ResponseContent;{{#operations}}{{#operation}}{{#allParams}}{{^isString}}{{^isUuid}}{{^isPrimitiveType}}{{^isContainer}}
+use crate::models::{{{dataType}}};{{/isContainer}}{{/isPrimitiveType}}{{/isUuid}}{{/isString}}{{/allParams}}{{/operation}}{{/operations}}
```

Comment:
I think the issue not imports itself, clients use clarified path to models, i.e `models::MyStruct`.

This change creates a lot of extra imports, for example affected apis by this PR - worked fine even before, but check/clippy will highlight that there are extra clarified paths.

Moreover now imports not unique, eg: https://github.com/OpenAPITools/openapi-generator/blob/65daa15704b73163193017e406b4a91f9842abbe/samples/client/petstore/rust/reqwest/petstore-async-middleware/src/apis/pet_api.rs#L15-L16 

I think the issue is this condition: `{{#isBodyParam}}models::{{/isBodyParam}}`  (e.g https://github.com/OpenAPITools/openapi-generator/blob/a8efb8eea8a8136311db2fa4dfea6df39d1c3d4a/modules/openapi-generator/src/main/resources/rust/reqwest/api.mustache#L93C483-L93C523)

So we have 2 questions:
1. why it's not true for these models?
2. can't we use `{{#isModel}}` instead? Like we do in `model.mustache`

Reply:
I've submitted this PR: 17968